src/praisonai-code/praisonai_code/runtime/server.py

```
@@ -124,6 +130,59 @@ def _evict_agent(self, key: str) -> None:
+        # Rehydrate prior history + wire persistence via the shared CLI helper.
+        # Imported lazily so the runtime module stays importable without the CLI
+        # state package and to keep cold-start cost off the anonymous path.
+        try:
+            from ..cli.state.project_sessions import apply_cli_session_continuity
+
+            apply_cli_session_continuity(agent, session_id, auto_save=session_id)
+        except Exception:
+            # If wiring fails, fall back to a fresh warm agent for this session
+            # rather than dropping the request; the anonymous contract (no
+            # persistence) applies until the store becomes available again.
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)